frontend/src/views/Login.vue

```diff
@@ -95,7 +95,9 @@ export default {
     a-form(:form="form" @submit="submitLogin")
       h1 Log in to your account
       a-form-item(v-if="$route.query.registerSuccess")
-        a-alert(type="success" message="You are now registered!" description="Use your credentials to log in below.")
+        a-alert(type="success" message="You are now registered!" description="We have sent you a verification email. If you can't find it, please check your spam box.")
```

Comment:
When token is either wrong or expired, prepare a message to show when redirected to here